checker/tests/nullness/java21/FlowSwitch.java

```diff
@@ -10,6 +10,7 @@ public class FlowSwitch {
   void test0(Number n) {
     String s = null;
     switch (n) {
+      // :: warning: (nulltest.redundant)
       case null, default: {
         // TODO: this should issue a dereference of nullable error.
```

Comment:
Note how this is inconsistent with the TODO here. Maybe this should be two test cases, one where the case is redundant and one where the TODO applies?